README.md

```diff
@@ -20,5 +20,19 @@ Invite yourself to the community Slack and join `#documentation-contrib`
 
 https://ez-community-on-slack.herokuapp.com/
 
+## Building & preview
+To build and preview your changes locally, you will need Python along with it's package manager (`pip`).
```

Comment:
```suggestion
To build and preview your changes locally, you will need Python along with its package manager (`pip`).
```

Reply:
I was wrong all my life...

https://www.faa.gov/about/initiatives/plain_language/articles/its/#:~:text=It's%20is%20a%20contraction%2C%20meaning,%3A%20Its%20color%20is%20red.)

:open_mouth: